linera-core/src/client/mod.rs

```diff
@@ -273,9 +273,7 @@ impl<Env: Environment> Client<Env> {
             ));
         }
 
-        let _ = self.ensure_has_chain_description(chain_id).await?;
```

Comment:
Why do we no longer have to check that?

Reply:
I believe that any code path that wants a chain description will retrieve it by calling `ensure_has_chain_description`.  So we should still check it, just lazily.